src/overlays/ovl_i4/fox_ka.c

```diff
@@ -1,63 +1,2183 @@
+            if (boss->swork[2] & 1) {
+                RCP_SetupDL(&gMasterDisp, 30);
+            }
+
+            if (boss->swork[12] <= 0) {
+                RCP_SetupDL(&gMasterDisp, 57);
+                gSPClearGeometryMode(gMasterDisp++, G_CULL_BACK);
+                *dList = &D_600D4E0;
+            }
+            break;
+
+        case 4:
+            rot->y -= boss->fwork[3];
+            if (boss->swork[3] & 1) {
+                RCP_SetupDL(&gMasterDisp, 30);
+            }
+
+            if (boss->swork[13] <= 0) {
+                RCP_SetupDL(&gMasterDisp, 57);
+                gSPClearGeometryMode(gMasterDisp++, G_CULL_BACK);
+                *dList = &D_600D290;
+            }
+            break;
+
+        case 9:
+            pos->x -= boss->fwork[4];
+            if (boss->swork[4] & 1) {
+                RCP_SetupDL(&gMasterDisp, 30);
+            }
+
+            if (boss->swork[14] <= 0) {
+                RCP_SetupDL(&gMasterDisp, 57);
+                gSPClearGeometryMode(gMasterDisp++, G_CULL_BACK);
+                *dList = &D_600D090;
+            }
+            break;
+
+        case 10:
+            if (boss->health < 0) {
+                RCP_SetupDL(&gMasterDisp, 57);
+                gSPClearGeometryMode(gMasterDisp++, G_CULL_BACK);
+                *dList = &D_6007300;
+            }
+            break;
+    }
+    return 0;
+}
+
+void func_i4_801968F4(Boss* boss) {
+    Vec3f jointTable[30];
+
+    if (boss->state != 0) {
+        if (boss->state < 20) {
+            gSPFogPosition(gMasterDisp++, gFogNear, 1002);
+        } else {
+            gSPFogPosition(gMasterDisp++, gFogNear, 1006);
+        }
+
+        Matrix_RotateY(gGfxMatrix, boss->fwork[13] * M_DTOR, 1);
+        Animation_GetFrameData(&D_60105D8, 0, jointTable);
+        Animation_DrawSkeleton(1, &D_6010744, jointTable, func_i4_801965A8, NULL, boss, &gIdentityMatrix);
+
+        gSPFogPosition(gMasterDisp++, gFogNear, gFogFar);
+        if (boss->fwork[14] > 0.0f) {
+            RCP_SetupDL(&gMasterDisp, 67);
+            Matrix_Push(&gGfxMatrix);
+            gDPSetPrimColor(gMasterDisp++, 0, 0, 255, 255, 255, 255);
+            gDPSetEnvColor(gMasterDisp++, 0, 255, 255, 255);
+            Matrix_Translate(gGfxMatrix, 0.0f, 500.0f, 0.0f, 1);
+            Matrix_Scale(gGfxMatrix, boss->fwork[14], boss->fwork[14], boss->fwork[14], 1);
+            Matrix_RotateX(gGfxMatrix, -(M_DTOR * 90), 1);
+            Matrix_SetGfxMtx(&gMasterDisp);
+            gSPDisplayList(gMasterDisp++, D_1024AC0);
+            Matrix_Pop(&gGfxMatrix);
+            Matrix_Push(&gGfxMatrix);
+            gDPSetPrimColor(gMasterDisp++, 0, 0, 255, 255, 255, 64);
+            gDPSetEnvColor(gMasterDisp++, 0, 255, 255, 64);
+            Matrix_Translate(gGfxMatrix, 0.0f, 500.0f, 0.0f, 1);
+            Matrix_Scale(gGfxMatrix, boss->fwork[14] * 3.0f, boss->fwork[14] * 3.0f, boss->fwork[14] * 3.0f, 1);
+            Matrix_RotateX(gGfxMatrix, -(M_DTOR * 90), 1);
```

Comment:
```suggestion
            Matrix_RotateX(gGfxMatrix, -90 * M_DTOR, 1);
```
I'd prefer to have these in the form `(degree angle) * M_DTOR`